shell/client/shell.html

```diff
@@ -1158,97 +1158,105 @@ <h3 id="app-stats">Apps</h3>
 
 <template name="referrals">
   {{#if quotaEnabled}}
+  {{>title "Referral Program"}}
   {{setDocumentTitle}}
-  {{>title "About Sandstorm"}}
-  <div class="app-details">
-    {{#if isPaid}}
-    <h1>Get up to 30GB of free space when your friends share on Oasis!</h1>
-    {{else}}
-    <h1>Get unlimited grains &amp; up to 2GB of free space when
-      your friends share on Oasis!</h1>
-    {{/if}}
-
-    <h2>Your reward</h2>
-
-    {{#if isPaid}}
-    <p>
-      You can earn <strong> up to 30GB of free storage!</strong> For
-      each friend that creates a sharing link, we'll reward you with 2
-      GB.
-    </p>
-    {{else}}
-    <p>You can earn <strong>unlimited grains and up to 2 GB of free storage!</strong></p>
-
-    <p>
-      For each friend that creates a sharing link, we'll reward you
-      with unlimited grains and 50 MB of free storage.  Tell them to
-      share an Etherpad grain or a Game of Y game with you.
-    </p>
-
-    {{/if}}
-
-    <h2>Sharing is referring</h2>
-
-    <ol>
-      <li>You share a grain with your friend (who isn't already an Oasis user).</li>
-      <li>Your friend signs in.</li>
-      <li>Your friend shares a grain with any other person (including you).</li>
-    </ol>
+  <div class="referrals">
+    <h1>Referral Program</h1>
+    <div class="pageheader">
+      <div class="leftbackground">
```

Comment:
These nested divs don't look like they're accomplishing anything. You can set the two backgrounds on the "pageheader" element and remove "leftbackground", "rightbackground", and "stoprotating".